### PR TITLE
Update device-health-status.md

### PR DESCRIPTION
**Pull request by boobooy2k**
'statue' changed to 'status'

### CODE REVIEW

File: Teams/alerts/device-health-status.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ Before you start, you'll need the teams/channel creation permissions in your ten
 |**Rule type**   |The device state rule helps you effectively manage. Teams devices and is classified as a device management type. In the future, more rules of device management type will be available to monitor other related capabilities (examples may include: unhealthy device and the sign-in status of device).|
```
Comment (JohanFreelancer9) [inline Teams/alerts/device-health-status.md:44]:
```suggestion
|**Rule type**   |The device state rule helps you effectively manage Teams devices and is classified as a device management type. In the future, more rules of device management type will be available to monitor other related capabilities (examples may include: unhealthy device and the sign-in status of the device).|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @boobooy2k, kindly commit the suggested change to fic the additional typos. Many thanks.

**Review by boobooy2k — COMMENTED** [review]
Please merge.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by cazawideh** [comment]
Reassigning to @dstrome, who is the writer for Teams Rooms and devices topics

**Pull request closed without merging**